src/app/zones/locales/en-us/index.yaml

```diff
@@ -76,7 +76,7 @@ zones:
         description: 'Establish a connection to your remote zone by following the instructions to setup dependencies and install the remote zone control plane.'
       scanner:
         title: 'Waiting for Zone to be connected …'
-        description: ' '
+        description: ''
```

Comment:
I noticed this from a while back, we changed this so you no longer need to use `' '` so I updated this while I was here.